### ISSUE TITLE
AccName-AAM: Include title attribute (as part of description computation) in Text Alternative Computation

### ISSUE THREAD

**Issue by devarship** [w3c/aria#584]
Reference: http://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_te

Purpose: Consider including title attribute as part of the accessible description calculation besides aria-describedby.

Rationale Follows (test environment: Win 7, JAWS 17, NVDA 2017, IE 11):
  
When using title attribute with valid label, label becomes the acc name and title gets appended to the label as acc description, with the output (Label [acc name] -> title). This is similar to when label and aria-describedby are used where the accumulated text is in the same order as when title was present, but replaced with the IDREF of aria-describedby (Label [acc name] -> aria-describedby [IDREF]). 

However, when both aria-describedby and title are present, aria-describedby takes precedence. Both JAWS and NVDA support the finding.

Since both NVDA and JAWS treat title as additive and similar to aria-describedby, I think the following sections should be updated:

4.2 Description Computation (http://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_description)-- include title attribute besides aria-describedby.

4.3.1 Terminology (http://www.w3.org/TR/accname-aam-1.1/#terminology) - under 2 B, a third bullet should be added to establish precedence level when title and aria-describedby are present.

thanks,
Devarshi

**Comment by jnurthen** [comment]
This issue was moved to w3c/accname#24
